### DISCUSSION TITLE
Mail sending going in Pending without any activity after enabling the ip pool

### DISCUSSION THREAD

**Discussion by jaydeepnp** [postalserver/postal#2570] [Help with using Postal] [answered]
Hi Postal Community,

Huge thanks to all postal community for great product. 

I need urgent help.

After enabling IP Pool with 2 IPs in  pool, mail mails goes in pending state without any activity on it. What can  be wrong.

Regards,
Jaydeep

**Comment by willpower232** [comment]
If email stays in pending, it usually means the workers aren't running so check everything is "up" according to `postal status`

**Reply by jaydeepnp** [reply]
Hi Will,

All services looks UP

Name                     Command               State   Ports
------------------------------------------------------------------
postal_cron_1       /docker-entrypoint.sh post ...   Up
postal_requeuer_1   /docker-entrypoint.sh post ...   Up
postal_smtp_1       /docker-entrypoint.sh post ...   Up
postal_web_1        /docker-entrypoint.sh post ...   Up
postal_worker_1     /docker-entrypoint.sh post ...   Up
[root@stagsmtp config]# postal logs worker
Attaching to postal_worker_1
worker_1    | [8] [2023-08-03T09:03:09.032] INFO -- : Worker running with 4 threads
worker_1    | [8] [2023-08-03T09:03:09.041] INFO -- : Joined main queue
worker_1    | [8] [2023-08-03T09:03:09.042] INFO -- : [RDRS1SZUHU] Started processing RequeueWebhooksJob job
worker_1    | [8] [2023-08-03T09:03:09.042] INFO -- : [MIFN7O8T3Z] Started processing SendNotificationsJob job
worker_1    | [8] [2023-08-03T09:03:09.139] INFO -- : [RDRS1SZUHU] Finished processing RequeueWebhooksJob job in 0.097589583s
worker_1    | [8] [2023-08-03T09:03:09.226] INFO -- : [MIFN7O8T3Z] Finished processing SendNotificationsJob job in 0.184113392s

But still no emails are getting processed and remains pending without any activity.

Regards,
Jaydeep

**Reply by willpower232** [reply]
The worker would have failed if rabbitmq wasn't available so it looks like everything is working.

You could double check the database and make sure the messages weren't "locked".

I guess the only remaining issue could be that the IP addresses you entered into the UI aren't available to the workers so they can't bind to them maybe?

**Reply by jaydeepnp** [reply]
Will,

Thanks for reply,

How I can check If all IPs in Pool are available to workers or not?

Regards,
Jaydeep

**Reply by willpower232** [reply]
if you can see them in `ip a` on the host server and you're running the container with host networking then that should be enough

if you aren't using host networking then I'm not sure what to suggest.

**Reply by jaydeepnp** [reply, marked as answer]
Will,

Thanks for reply , got clue from reply.

It resolved my issue by using  IP from system (private).

Regards,
Jaydeep
